app/main.py

```diff
@@ -1,8 +1,21 @@
+
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    list_person = [Person(person["name"], person["age"]) for person in people]
+    for person in people:
+        if "wife" in person and person["wife"] is not None:
+            his_wife = Person.people[person["wife"]]
+            Person.people[person["name"]].wife = his_wife
+        if "husband" in person and person["husband"] is not None:
+            her_husband = Person.people[person["husband"]]
+            Person.people[person["name"]].husband = her_husband
```

Comment:
You could retrieve the `Person.people[person["name"]]` at the beginning of the loop and write it to a variable for further use to avoid code duplication here and on line 16